.eslintrc

```diff
@@ -9,7 +9,7 @@
   ],
 
   "parserOptions": {
-    sourceType": "module",
+    "sourceType": "module",
```

Comment:
No, it was not erroring at any place that I have seen. Once I configured lint I thought it was not running due to that, but seems it is not the case.